<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnclosureWidget</class>
 <widget class="QWidget" name="EnclosureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1433</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="m_pMenu">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>140</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>140</height>
      </size>
     </property>
     <widget class="ProjectMenu" name="m_pProjectMenu"/>
     <widget class="EnclosureMenu" name="m_pEnclosureMenu"/>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="chartsView" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EnclosureMenu</class>
   <extends>QWidget</extends>
   <header>enclosuremenu.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProjectMenu</class>
   <extends>QWidget</extends>
   <header>projectmenu.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
